### ISSUE TITLE
More realistic still recipe

### ISSUE THREAD

**Issue by PaulBlay** [CleverRaven/Cataclysm-DDA#47201]
# Is your feature request related to a problem? Please describe.

I think we can do better than the current still recipe, which is a garden hose in a cooking pot.

# Describe the solution you'd like

I think the following recipe could substitute for the one currently in 
json\recipes\tools\tool.json

```
  {
    "result": "still",
    "byproducts": [ [ "material_sand", 1 ] ],
    "type": "recipe",
    "activity_level": "MODERATE_EXERCISE",
    "category": "CC_OTHER",
    "subcategory": "CSC_OTHER_TOOLS",
    "skill_used": "fabrication",
    "skills_required": [ "chemistry", 3 ],
    "difficulty": 4,
    "time": "200 m",
    "book_learn": [ [ "textbook_chemistry", 4 ], [ "brewing_cookbook", 3 ] ],
    "qualities": [
      { "id": "HAMMER", "level": 2 } ,
      { "id": "DRILL", "level": 2 }
    ],
    "components": [ 
      [ [ "pot", 1 ], [ "pot_copper", 1 ] ], 
      [ [ "cu_pipe", 1 ] ], 
      [ [ "thermometer" , 1 ] ], 
      [ [ "bone_glue", 1 ], [ "superglue", 1 ] ] ,
      [ [ "duct_tape", 1 ] ],
      [ [ "material_sand", 1 ] ]
    ]
  },
```

# Describe alternatives you've considered

Leave as is.

# Additional context

References: 
* https://diyprojects.com/make-still-diy/
* https://www.instructables.com/How-to-bend-pipe-without-a-pipe-bender/

I've included 'sand' as a component and a byproduct, because I'm not sure how to code for a required component that isn't used up.

Also, 'copper_tubing' seems to be really short. Just three inches? And it's a 0.5L volume. That puts it at ... 3 inches long, 3.6 inches diameter (unless my maths is wrong). That's just crazy. [EDIT] Pipes are cubes now. See https://github.com/CleverRaven/Cataclysm-DDA/issues/47202 for what's going on and how to fix it. I have altered the recipe based on the idea that copper tubing is 1m (or 3 feet) long, not three inches.

**Comment by Zireael07** [comment]
Why does the still have a hammer quality? That's the only thing I'm not certain about (and the still recipe IIRC was kind of a random mash of things from the net)

**Comment by PaulBlay** [comment]
The hammer is used to bend the copper tubing.

**Comment by Zireael07** [comment]
Ah, whoops, I misread as giving the still a hammer quality. Requiring a hammer quality is fine although I know people who bend copper tubing (with sand inside) with bare hands and/or rocks (and they weren't strongmen by any definition of the word)

**Comment by Fosheze** [comment]
> Ah, whoops, I misread as giving the still a hammer quality. Requiring a hammer quality is fine although I know people who bend copper tubing (with sand inside) with bare hands and/or rocks (and they weren't strongmen by any definition of the word)

This depends on what we are talking about when it comes to copper tubing. I believe that in game it is supposed to represent the smaller and thinner walled tubing commonly used in low pressure hvac, pneumatics, and feed for low volume liquid plumbing (refrigerator water dispenser, drinking water taps, etc). That tubing can be bent by hand fairly easily using the sand method. If we are talking about actual copper plumbing pipe though you most likely aren't going to be bending that by hand. 

Also of course there's a [wiki page](https://en.wikipedia.org/wiki/Copper_tubing#:~:text=In%20the%20plumbing%20trade%20the,and%20108%20mm%20outside%20diameters.) for this.

**Comment by Tamiore** [comment]
"thermometer" currently can't be crafted or substituted for.  
Including it in the crafting recipe for a still as a requirement essentially renders that recipe largely moot unless you just happened to find proper factory-made thermometer.

**Comment by PaulBlay** [comment]
> "thermometer" currently can't be crafted or substituted for.

My character has five thermometers so it can't be a very rare drop. That said, you could make a craft recipe for a digital thermometer from an ultra-light battery, a bimetal thermostat, a small LCD screen, some electronics scrap and maybe a bit of cardboard and duct tape to wrap it all up.

**Comment by Tamiore** [comment]
> My character has five thermometers so it can't be a very rare drop. 
Besides the point. 
Still itself technically does not require a modern thermometer to be built. (Stills were being built before modern thermometers.) 
Thermometer belongs as part of the crafting recipe for a given distillation process — where it can accurately represent how crucial precise temperature control is for that specific distillation.

**Comment by PaulBlay** [comment]
> Besides the point.

Besides the point you're making now, not besides the point you were making in your original comment.

> Thermometer belongs as part of the crafting recipe for a given distillation process — where it can accurately represent how crucial precise temperature control is for that specific distillation.

In the site referenced, and by implication in the recipe I submitted, the thermometer is literally placed part way through a hole drilled in the lid and glued there. That makes it part of this still. 

At present no recipe that requires distilling or fine distilling also requires a thermometer.  Most of them should require a firm grip on how hot things are.  Now you could add thermometer to all those, however I think it makes more sense to include temperature control in the concept of what is required for providing a distilling quality of 2.

**Comment by Tamior** [comment]
>Besides the point you're making now, not besides the point you were making in your original comment.

The point I was making in my original comment is:
recipes should avoid inclusion of extra components as obligatory that are not mandatory for the basic function of the product.
Because in real life you can just forgo the inclusion of a thermometer (if you don't have one at hand) and still end up with an otherwise perfectly functional still. 

>In the site referenced, and by implication in the recipe I submitted, the thermometer is literally placed part way through a hole drilled in the lid and glued there. That makes it part of this still.

And in real life you can still make a functional still without a thermometer, and just leave that "hole" plugged for now.

>At present no recipe that requires distilling or fine distilling also requires a thermometer. 

Factually incorrect: http://cdda-trunk.chezzo.com/chem_ethanol/craft

>Most of them should require a firm grip on how hot things are.

There is a difference between "a grip on how hot things are" and "requires a factory-made thermometer".
Again, some distillation require high temperature control precision, yes. Some don't. 

In conclusion:
- given the level of abstraction the game operates at, it feels most appropriate for the still itself to have a "slot" for thermometer build into it (you can mention it in the description), but not require a thermometer to be there before the assembly of the still can even begin. 
- thermometer (or lack thereof) should be a requirement a given distillation recipe

**Comment by PaulBlay** [comment]
Submit a PR. You win because you got your preferred solution, I win because I get a solution.

**Comment by Tamior** [comment]
>Submit a PR. You win because you got your preferred solution, I win because I get a solution.

Your wish is my command, M'lord.

https://github.com/CleverRaven/Cataclysm-DDA/pull/47226

**Comment by casswedson** [comment]
This issue can be closed, fixed by #47201

**Comment by anothersimulacrum** [comment]
You mean #47226?
I agree.
